core/src/main/java/bio/terra/pearl/core/model/fileupload/ParticipantFileUpload.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.pearl.core.model.fileupload;
+
+import bio.terra.pearl.core.model.BaseEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class ParticipantFileUpload extends BaseEntity {
```

Comment:
I'd just as soon call this "participantFile". "ParticipantFileUpload" sounds more like it is a record fo the upload process, not a pointer to the file.  And then we can use this to track files that might not have come in via direct upload
```suggestion
public class ParticipantFile extends BaseEntity {
```